### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by justified1gmail** [Videogamers0/StoryManager#7] [Ideas]
Off the top of my head, here's a few ideas in no particular order:

1. Increase max story font size to 36 or remember zoom settings on reload
2. Add custom story margin sizes
3. Add selectable story font
4. Add context menu to retrieve stories by author from an author name at the top of a story
5. Add hotkey to open downloader
6. If the clipboard url includes memberpage.php then prefill the author url field with it

**Comment by Videogamers0** [comment]
Hi thanks for the feedback. I'll likely implement a few of these this weekend when I've some spare time

Hah, I didn't even notice you could zoom the story with ctrl+plus. And not sure why the font size isn't working. It was supposed to be an editable combobox so you could type in any value you wanted, but looks like it's only accepting values that are already listed in the dropdown.

**Comment by Videogamers0** [comment]
Implemented a few of these in [Version 1.0.4](https://github.com/Videogamers0/StoryManager/releases):

> 1. Increase max story font size to 36 or remember zoom settings on reload

6157e9bae386d65b12b7556f72bdfd5810f72314 You can now enter any value from 10 to 50 into the font size dropdown. 

> 3. Add selectable story font

05a5eb2c3a18484a02d65cf4c7c5eda7d472b681 The settings dialog now has a Font Family dropdown.

> 5. Add hotkey to open downloader

c808c04a4c9b12daf821a7ce1ce866c1db6737af Ctrl+D opens the downloader dialog.

> 6. If the clipboard url includes memberpage.php then prefill the author url field with it

84187c9efd68b14b89f5b7de0e123b905f061220

**Reply by justified1gmail** [reply]
Wow, this is fantastic.
